wordpress-mobile/WordPress-iOS
# Use podspecs directly from gutenberg-mobile

This uses the changes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/262 to fetch the React Native podspecs from the gutenberg-mobile repo instead of having them in this one.

To test:

- Run `pod install`
- Make sure the app still builds and runs properly
